docs/src/examples/components/Flex/Types/FlexExampleColumns.shorthand.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+import { Flex, Segment } from '@stardust-ui/react'
+
+const FlexExampleColumns = () => (
+  <>
+    <Flex gap="gap.small" padding="padding.medium">
```

Comment:
What is the reason for defining string options containing dot? At first I thought that `gap`, `padding` and `size` are some variables.

Reply:
@mnajdova because the prop values need to be unique through the app; @kuzhelov is matching their values with variables keys to achieve styling, see https://github.com/stardust-ui/react/pull/802#discussion_r254673039

Reply:
acknowledge the issue - yes, it was done exactly for the reasons @Bugaa92 has mentioned. However, indeed, we might do better here - let me address this moment. Thank you!